azure-iot-hub/azure/iot/hub/constant.py

```diff
@@ -6,4 +6,4 @@
 """This module defines constants for use across the azure-iot-hub package
 """
 
-VERSION = "2.2.0-pnp-preview-refresh.0"
+VERSION = "2.2.0-preview.0"
```

Comment:
I thought it had to be RC 1 or something like that.

Reply:
it converts to rc once u produce wheel.